pkg/pipelines/tekton/pac/pac.go

```diff
@@ -21,12 +21,15 @@ const (
 	routePacLabel          = "pipelines-as-code/route=controller"
 )
 
+// Namespaces where PaC is commonly installed.
+var usualSuspects = []string{"pipelines-as-code", "openshift-pipelines"}
+
 // DetectPACInstallation checks whether PAC is installed on the cluster
 // Taken and slightly modified from https://github.com/openshift-pipelines/pipelines-as-code/blob/6a7f043f9bb51d04ab729505b26446695595df1f/pkg/cmd/tknpac/bootstrap/bootstrap.go
-func DetectPACInstallation(ctx context.Context, wantedNamespace string) (bool, string, error) {
```

Comment:
Does this break things? (removing the ns as the argument)

Reply:
Before it was always an empty string so it was effectively unused.